examples/README.md

```diff
@@ -6,16 +6,18 @@ The `ballerinax/aws.redshift` connector facilitates seamless integration with Am
 
 2. [Insert data in to the database](https://github.com/ballerina-platform/module-ballerinax-aws.redshift/blob/main/examples/execute) - Connects to AWS Redshift using the Redshift connector and performs an INSERT operation into a specified table
 
+3. [Music store](https://github.com/ballerina-platform/module-ballerinax-aws.redshift/blob/main/examples/music-store) -  This example illustrates the process of creating an HTTP RESTful API with Ballerina to perform basic CRUD operations on a database, specifically AWS Redshift, involving setup, configuration, and running examples.
```

Comment:
Better to include these introduction in the Module.md/README documentation. Else the ##Examples heading includes only one sentence